src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    static final int NUMBER_OF_BALLS = 3;

```

Comment:
```suggestion
    private static final int NUMBER_OF_BALLS = 3;

```
better move this constant to the Application class